wallet-new/integration/TransactionSpecs.hs

```diff
@@ -21,6 +20,7 @@ import qualified Pos.Core as Core
 import           Pos.Util.Log.LoggerConfig (defaultTestConfiguration)
 import           Pos.Util.Wlog (Severity (Debug), setupLogging)
 
+import           AccountSpecs (noLongerThan, oneSecond, pollTransactions)
```

Comment:
This feels like those functions should be defined and accessible from somewhere else.